src/routes/organization/invite/[inviteId]/+page.server.ts

```diff
@@ -0,0 +1,66 @@
+import { checkSessionAuth, deletePath, pushPath, readPath, writePath } from '$lib/server/firebaseUtils.js';
+import { error, redirect } from '@sveltejs/kit';
+import type { PageServerLoad, Actions } from './$types.js';
+import firebaseAdmin from 'firebase-admin';
+import firebaseAdminCredential, { databaseURL } from "$lib/server/firebaseAdminCredential";
+
+if (!firebaseAdmin.apps.length) {
+  firebaseAdmin.initializeApp({
+    credential: firebaseAdmin.credential.cert(firebaseAdminCredential),
+    databaseURL
+  });
+}
```

Comment:
We don't need this snippet, it gets done in the utils library now, and it's important that it just happens in one place because not the config can change based on environment